html_css/v2/foundations/positioning-and-layout/positioning-and-layout.md

```diff
@@ -0,0 +1,77 @@
+
+### Introduction
+Now that you understand the basic syntax of HTML and CSS, we're going to get serious. The most important skills you need to master with CSS are _positioning_ and _layout_. Changing fonts and colors is a crucial skill, but being able to put things exactly where you want them on a webpage is even more so. After all, how may webpages can you find where absolutely every element is just stacked one on top of another?
+
+### Learning Outcomes
+* You'll learn about "Normal flow"
+* You'll learn all about _the box model_.
+* You'll learn how to make sure elements are just the right size with `margin`, `padding`, and `borders`
+* You'll learn the difference between `block` and `inline` elements.
+
+Learning to position elements on a webpage is not that difficult once you understand just a few key concepts. Unfortunately many learners race through learning HTML and CSS to get to JavaScript and end up missing these fundamental concepts. This leads to frustration, pain ([and funny gifs](https://giphy.com/gifs/css-13FrpeVH09Zrb2)), because all of the JavaScript skills in the world are meaningless if you can't stick your elements on the page where you need them to be. So with that in mind, let's get started.
+
+### The Box Model
+The first important concept that you need to understand to be successful in CSS is the box model. It isn't complicated, but skipping over it now will cause you much frustration down the line.
+
+Basically, every single thing on a webpage is a rectangular box. These boxes can have other boxes in them and can sit alongside one another. You can get a rough idea of how this works by sticking a border on every item on the page like this:
+
+~~~css
+* {
+  border: 2px solid red
+}
+~~~
+
+![boxes](./imgs/boxes.png)
+
+You can use the browser's inspector to add this css to this webpage if you want. Boxes in boxes!
+
+![lines](./imgs/odin-lined.png)
+
+When it comes down to it, laying out a webpage and positioning all of it's elements is basically just deciding how you are going to nest and stack these boxes.
+
+The only real complication here is that there are multiple ways to manipulate the size of these boxes, and the space between them using `margin`, `padding`, and `border`. The assigned articles go into more depth on this concept, but to sum it up briefly: padding increases the space between the edge of a box and the content inside of it, margin increases the space between a box and any others that sit next to it, and border adds space (even if it's only a pixel or two) between the margin and the padding. Be sure to study the diagrams carefully.
+
+![the box model](./imgs/box-model.png)
+
+<div class="lesson-content__panel" markdown="1">
```

Comment:
Are we going to have 2 assignments in this lesson? the side nav on lesson pages may have a hissy fit if we do 😆

Reply:
I think the side nav just looks for headings.... and there's no heading here.

I could be wrong about that tho

Reply:
You're right it does, I still think we should be consistent with 1 assignment box per lesson. But we can come back to that  when we start putting everything together.